examples/chef/devices/rootnode_colortemperaturelight_hbUnzYVeyn.zap

```diff
@@ -3813,7 +3453,7 @@
               "storageOption": "RAM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0x01C6",
+              "defaultValue": "0xFEFF",
```

Comment:
This is wrong, previous value was correct

Reply:
ColorTempPhysicalMaxMireds in the spec says max is 65279

Reply:
Yes. And 65279 = 0xFEFF

![image](https://github.com/user-attachments/assets/ba6d7851-89ad-4aa5-98f3-b9fd494d3911)